exercises/practice/nth-prime/t/nth-prime.rakutest

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env raku
+use Test;
+use lib $?FILE.IO.parent(2).add('lib');
+use NthPrime;
+
+for Int, Rat, FatRat {
+  .^method_table<is-prime>.wrap: method (|) {
+    bail-out 'Access to the built-in is-prime method has been disabled for this exercise.';
```

Comment:
I like it, the tests are meant to be read and states clearly what the README/Instructions state less clearly.